generators/ci-cd/templates/azure-pipelines.yml.ejs

```diff
@@ -119,6 +132,12 @@ jobs:
   <%= commented %>    CYPRESS_ENABLE_RECORD: <%= cicdIntegrationsCypressDashboard %>
   <%= commented %>    CYPRESS_PROJECT_ID: $(CYPRESS_PROJECT_ID)
   <%= commented %>    CYPRESS_RECORD_KEY: $(CYPRESS_RECORD_KEY)
+  <%= commented %>- task: PublishPipelineArtifact@1
+  <%= commented %>  inputs:
+  <%= commented %>    artifactName: 'cypress-screenshots'
+  <%= commented %>    targetPath: '$(System.DefaultWorkingDirectory)/build/cypress/screenshots'
```

Comment:
```suggestion
  <%= commented %>    targetPath: '$(System.DefaultWorkingDirectory)/*/cypress/screenshots'
```
maven uses target

Reply:
@mshima Thanks for your suggestion. Unfortunately this does not work for all paths (Path does not exist error).

I use Gradle as build system and did not thought about Maven which would have `target`  as output directory.

How about declaring a variable on top of the Pipeline and using it for all paths?

```yaml
variables:
<%_ if (buildToolMaven) { _%>
  buildDirectory: $(Build.SourcesDirectory)/target
<%_ } else if (buildToolGradle) { _%>
  buildDirectory: $(Build.SourcesDirectory)/build
<%_ } _%>
```

Reply:
And now? Using DIST_DIR?

Reply:
If we use the variables from the pipeline we could then use that variable for all paths:

```yaml
targetPath: '$(outputDirectory)/cypress/screenshots'
```

If we use your approach, then `BUILD_DIR` would be the correct variable.

We could also combine that to:

```yaml
variables:
  outputDirectory: $(Build.SourcesDirectory)/<%= BUILD_DIR %>

# ... and use it ...

targetPath: '$(outputDirectory)/cypress/screenshots'
```

Which one do you prefer?

Reply:
Sorry, my suggestion was wrong.
The var I was looking for is buildDir.
It should have target or build value.

If you think it’s better to extract the variable, ok to me.

Reply:
Your choice.
BUILD_DIR is correct.